icons/layers-3.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m6 14.5-3.4 1.7a.83.83 0 0 0 0 1.6l7.6 3.8a4.14 4.14 0 0 0 3.58 0l7.62-3.8a.83.83 0 0 0 0-1.6L18 14.5" />
+  <path d="m6 9.5-3.4 1.7a.83.83 0 0 0 0 1.6l7.6 3.8a4.14 4.14 0 0 0 3.58 0l7.62-3.8a.83.83 0 0 0 0-1.6L18 9.5" />
+  <path d="M13.8 2.4a4.14 4.14 0 0 0-3.58 0L2.6 6.2a.83.83 0 0 0 0 1.6l7.6 3.8a4.14 4.14 0 0 0 3.58 0l7.62-3.8a.83.83 0 0 0 0-1.6Z" />
```

Comment:
What about using this as `layers`, refine the one we have instead of creating variants.

Reply:
> What about using this as `layers`, refine the one we have instead of creating variants.

@jguddas I’d be fine with that, but I think @karsa-mistmere want’s to keep the existing `layers` around, based on https://github.com/lucide-icons/lucide/pull/1142#discussion_r1196963327 …

> Not really a fan of changing the essence of `layers` so drastically. 😕

Reply:
Also has wonky corners:
![image](https://github.com/lucide-icons/lucide/assets/17746067/5251e4ca-077d-4e40-a4a1-cd6a0125dbf7)

```suggestion
  <path d="m12.83 2.18a2 2 0 0 0-1.66 0L2.6 6.08a1 1 0 0 0 0 1.83l8.58 3.91a2 2 0 0 0 1.66 0l8.58-3.9a1 1 0 0 0 0-1.83Z" />
  <path d="m6.08 9.5-3.5 1.6a1 1 0 0 0 0 1.81l8.6 3.91a2 2 0 0 0 1.65 0l8.58-3.9a1 1 0 0 0 0-1.83l-3.5-1.59" />
  <path d="m6.08 14.5-3.5 1.6a1 1 0 0 0 0 1.81l8.6 3.91a2 2 0 0 0 1.65 0l8.58-3.9a1 1 0 0 0 0-1.83l-3.5-1.59" />
```